rocky/assets/css/components/scan-level-indicator.scss

```diff
@@ -17,6 +17,10 @@ Markup:
   gap: 0.25rem;
   background-color: transparent;
 
+  &.left {
+    padding: 0;
+  }
```

Comment:
What is the reason this is changed here, beause as far as I can see this isn't used in the reporting templates?

Reply:
Yes, I think we can remove this and make it a generic thing, as the cat paws are all left aligned.

Reply:
This could be set on the level indicator itself, they are through the rest of rocky left aligned, no paddings.